client/webserver/site/src/js/markets.js

```diff
@@ -535,17 +535,20 @@ export default class MarketsPage extends BasePage {
       window.clearTimeout(this.preorderTimer)
       this.preorderTimer = null
     }
+    const mktId = marketID(baseCfg.symbol, quoteCfg.symbol)
     this.market = {
       dex: dex,
-      sid: marketID(baseCfg.symbol, quoteCfg.symbol), // A string market identifier used by the DEX.
+      sid: mktId, // A string market identifier used by the DEX.
       // app.assets is a map of core.SupportedAsset type, which can be found at
       // client/core/types.go.
       base: app.assets[base],
       quote: app.assets[quote],
+      lotSize: dex.markets[mktId].lotsize,
+      rateStep: dex.markets[mktId].ratestep,
```

Comment:
Done, although I make the field `cfg` because `market.mktCfg` stuttered.  Anyway, the market config properties might be of use so the whole cfg is good to have.